opendevin/runtime/docker/image_agnostic_util.py

```diff
@@ -61,8 +61,8 @@ def _build_sandbox_image(
 
 
 def _get_new_image_name(base_image: str) -> str:
-    if ":" not in base_image:
-        base_image = base_image + ":latest"
+    if ':' not in base_image:
+        base_image = base_image + ':main'
```

Comment:
This is not right. When a user provides an image without a tag, it should automatically choose the latest version, not the main version. This can cause confusion.

Reply:
For example, set `sandbox_container_image=debian` actually uses `debian:latest`, not `debian:main`.